src/app/app.module.ts

```diff
@@ -1,18 +1,18 @@
+//Core imports
```

Comment:
I  re worked the import structure and actually imported chat service dependencies.